test/gui/shared/scripts/helpers/api/oc10.py

```diff
@@ -50,13 +51,24 @@ def create_user(username, password, displayname, email):
         "email": email,
     }
     response = request.post(url, body)
-    request.assertHttpStatus(response, 200)
+    if response.status_code != 200:
+        root = ET.fromstring(response.content)
+        message = root.find(".//message")
+        raise Exception(
+            "Creating user '%s' failed with %s : " % (username, response.status_code)
+            + message.text
+        )
```

Comment:
or just print the response content?
```suggestion
            "Creating user '%s' failed with %s : \n" % (username, response.status_code)
            + response.content
        )
```